addons/pos_sale/models/pos_order.py

```diff
@@ -163,6 +163,11 @@ def _get_invoice_lines_values(self, line_values, pos_line):
 
         return inv_line_vals
 
+    def write(self, vals):
+        if 'crm_team_id' in vals:
+            vals['crm_team_id'] = vals['crm_team_id'] if vals.get('crm_team_id') else self.session_id.crm_team_id.id
```

Comment:
`self.session_id.crm_team_id.id` is always defined ?

Reply:
@davidmonnom No, but if you don't set a Sales team `self.session_id.crm_team_id.id` would be `False`, and it's correct.